### DISCUSSION TITLE
Telegram topics support

### DISCUSSION THREAD

**Discussion by darkymehdi** [alexjustesen/speedtest-tracker#1405] [Ideas]
Is there a way to configure telegram notification in a specific topic ? Or is it planned ? :) Thanks

**Comment by alexjustesen** [comment]
@mehdimansouriut can you share a link to their docs so I can take a look?

**Comment by darkymehdi** [comment]
https://core.telegram.org/api/forum

Here is the doc :) Thank you for your answer

**Reply by alexjustesen** [reply]
Oh so you're looking to create a forum post with a topic?

**Reply by darkymehdi** [reply]
I'm not sure if this is the good term but I have a group chat where I enabled the "Topics" option. The group chat has an Id and the topics wich is like a subchat as an ID too. and for other services when I give the chat id It's like that : -chat_Id/topic_Id

**Reply by alexjustesen** [reply]
Ok cool, I don't use telegram much so I wanted to understand the use case.

**Reply by darkymehdi** [reply]
Well if you want the complete use case aha : I have a lot of self hosted services like plex, radarr, sonarr etc ans they all send notifications. And I prefer using topivs than putting all the notifications in the same chan :)
